bin/linters.sh

```diff
@@ -37,7 +38,7 @@ go_metalinter() {
 
     if [[ -z ${LAST_GOOD_GITSHA} ]] && [[ -n "${parent_branch}" ]]; then
         LAST_GOOD_GITSHA="$(git log ${parent_branch}.. --pretty="%H"|tail -1)"
-        [[ ! -z ${LAST_GOOD_GITSHA} ]] && LAST_GOOD_GITSHA="${LAST_GOOD_GITSHA}^"
+        [[ ! -z ${LAST_GOOD_GITSHA} ]] && LAST_GOOD_GITSHA="${LAST_GOOD_GITSHA}"
```

Comment:
I'm not sure what this line means now. This line does not change `LAST_GOOD_GITSHA` anyways. Just removing this line would be better.

Reply:
Done